Tests/Clappr_Tests/Classes/Plugins/Core/MediaControl/MediaControlTests.swift

```diff
@@ -76,35 +78,41 @@ class MediaControlTests: QuickSpec {
 
             /*
              describe("#gestureRecognizer") {
-                it("toggles the viewcontroller's swipe to back gesture to true") {
-                    let viewController = UIViewControllerMock()
-                    let options: Options = [kMediaControlPlugins: [Seekbar.self]]
-                    let core = Core(loader: Loader(), options: options)
-                    let mediaControl = MediaControl(context: core)
-                    viewController.view.addSubview(mediaControl)
-                    mediaControl.render()
-                    if let seekbar = mediaControl.plugins.first(where: { $0.pluginName == Seekbar.name }) as? Seekbar {
-                        seekbar.view.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
-                    }
-                    
-                    _ = mediaControl.gestureRecognizer(UITapGestureRecognizer(), shouldReceive: UITouch())
-                    
-                    expect(viewController.navigationController?.interactivePopGestureRecognizer?.isEnabled).to(beFalse())
-                }
-            }
+             it("toggles the viewcontroller's swipe to back gesture to true") {
```

Comment:
Why is this commented?

Reply:
Ok, but I highly suggest to not ever leave commented code behind.
We are very likely to forget it later due to priority changing.

Reply:
I was looking that one of the tests was deleted and the other one commented. Why the different decision?

Reply:
Sure. We remove this part of the code. Thanks =]